### PR TITLE
README: minor fixes

### PR DESCRIPTION
**Pull request by igoropaniuk**
Add additional pre-requisites (needed for folder sharing in QEMU/QEMUv8 + make and xterm)
Minor fixes of table of contents

`Signed-off-by: Igor Opaniuk <igor.opaniuk@linaro.org>`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,7 +122,8 @@ $ sudo apt-get install android-tools-adb android-tools-fastboot autoconf bc \
 	bison cscope curl flex gdisk libc6:i386 libfdt-dev libftdi-dev \
 	libglib2.0-dev libhidapi-dev libncurses5-dev libpixman-1-dev \
 	libstdc++6:i386 libtool libz1:i386 mtools netcat python-crypto \
-	python-serial python-wand unzip uuid-dev xdg-utils xz-utils zlib1g-dev
+	python-serial python-wand unzip uuid-dev xdg-utils xz-utils zlib1g-dev \
+	xterm make libcap-dev libattr1-dev
```
Comment (jbech-linaro) [inline README.md]: The list her has been sorted in alphabetic order to make it easy to see if something already is there. Please put the new dependencies in alphabetic order too.

**Comment by jbech-linaro** [comment]
Just a single comment, other than that:
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

**Comment by igoropaniuk** [comment]
BTW, just found (setup based on ubuntu 16.04) that repo should added also. After installing all pre-requisites, still receive:
```
xdev@opteebuilder-vm:/opt/linaro/rpi3$ repo init -u https://github.com/OP-TEE/manifest.git -m rpi3.xml
The program 'repo' can be found in the following packages:
 * phablet-tools
 * repo
```

**Comment by jbech-linaro** [comment]
Hmm, maybe. If it is found on the major distros, well, why not. Otherwise we have a separate row in the README.md telling how to get repo.
https://github.com/OP-TEE/optee_os#51-install-repo

But, in any case, it shouldn't hurt to have repo there too. Feel free to add it if you like.

**Comment by igoropaniuk** [comment]
Sorry, I haven't seen this section about installing repo
I think it isn't needed to add it to the list of pre-requisites, because it's already mentioned in this doc.

**Comment by igoropaniuk** [comment]
I think I've addressed all comments so far. Does anybody have something more to mention?

**Comment by jforissier** [comment]
@igoropaniuk can you please rebase?

**Comment by igoropaniuk** [comment]
@jforissier done

**Pull request merged**